src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,52 @@
-import React from 'react';
+/* eslint-disable max-len */
+
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard = ({ movie }: Props) => (
```

Comment:
In [React Cheat Sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript) it is suggested to use React.FC to type functional components, I personally don't agree with this practice -> [here's why](https://github.com/facebook/create-react-app/pull/8177) 
But I'm wondering why you didn't use it?

Reply:
I didn't use that because. Firstly it's just more code to write. Secondly TypeScript's type inference. Thirdly I prefer ES6 syntax when I assign default value to prop.